gesturelibrary/src/double.c

```diff
@@ -52,21 +69,23 @@ static void update_double_taps(double_tap_t* double_tap, stroke_t* stroke, tap_t
                 break;
             }
         }
+        if (on_double_tap) {
```

Comment:
It is possible to break out of the switch statement BEFORE invoking this listener. Since there are state changes before this invocation, should this call be placed before the break statements?

Reply:
Do you even need the break statements? Do you not want to report a double tap regardless of if the state changes to failed or completed or stays as in-progress?